src/app/api/shifts/[id]/route.ts

```
@@ -21,12 +21,35 @@ export async function GET(
   try {
     // First, get the shift
     const shift = await prisma.shift.findUnique({
+      where: {
+        id: params.id,
+      },
```

Biome (1.9.4)

[error] This property value named where is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named where


(lint/suspicious/noDuplicateObjectKeys)